pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "mm_classification"
+version = "0.1.0"
+packages = []
```

Comment:
This should probably explicitly list `mm_classification`

Reply:
added this